plugins/terminal.koplugin/main.lua

```diff
@@ -86,9 +101,19 @@ function Terminal:spawnShell(cols, rows)
 
     if C.grantpt(self.ptmx) ~= 0 then
         logger.err("Terminal: can not grantpt")
+        C.close(self.ptmx)
+        UIManager:show(InfoMessage:new{
+            text = _("Your firmware doesn't support the 'grantpt' system call.\nTerminal can not be used."),
```

Comment:
Maybe just a logger.err without translation? I'm not a fan from the translator's perspective.

Reply:
Yes, maybe a logger msg would be enough. This message shouldn't pop up anyway, as we disable the plugin now (with this PR) if grantpt is does not work.